dtu_mlops_project/models/predict_model.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import dotenv
+import torchaudio
+import torch
+import hydra
+import numpy as np
+
+from dtu_mlops_project.models.model import SpeechSpectrogramsTransferLearning
+
+dotenv.load_dotenv(override=True)
+
+
+class PreprocPlusModel(torch.nn.Module):
```

Comment:
@miccio-dk perhaps we can add functionality for the class to either accept a [1, 16000] array, or a str that is a filepath to a .wav file, in which case it will first load the .wav file and compute the [1, 16000] array. Would that be most convenient to be able to both work with data from speechcommands dataset as well as new audio files?

Reply:
we already have that, it's the forward() and wav_to_yhat() functions respectively